argoexec/rockcraft.yaml

```diff
@@ -65,30 +52,25 @@ parts:
       - mailcap
       - procps
       - tar
-      - libcap2-bin
       - libcap2-dev
       - jq
-      - zip
     stage-packages:
       - git
       - libcap2
       - jq
       - tar
       - gzip
     override-build: |
-      # builder stage https://github.com/argoproj/argo-workflows/blob/v3.3.10/Dockerfile#L36
+      # builder stage https://github.com/argoproj/argo-workflows/blob/v3.4.16/Dockerfile#L36
       go mod download
 
       # reset LDFLAGS because Go has different format, i.e. '-L' is undefined in Go
       LDFLAGS=""
 
-      # argoexec-build stage https://github.com/argoproj/argo-workflows/blob/v3.3.10/Dockerfile#L82
-      cat .dockerignore >> .gitignore
-      git status --porcelain | cut -c4- | xargs git update-index --skip-worktree
+      # argoexec-build stage https://github.com/argoproj/argo-workflows/blob/v3.4.16/Dockerfile#L82
       make dist/argoexec
```

Comment:
upstream is now passing GIT information to this make command ... do we want to include it ? https://github.com/argoproj/argo-workflows/blob/v3.4.16/Dockerfile#L52C91-L52C95

Reply:
I also saw it, but the git information comes from their repository instead of ours. Do you think we need it? FWIW, in a previous version of this rock we also slightly changed that command.

Reply:
ok I checked the source code ... looks like they use just to show to user which version of workflow is being used .... I think we may skip it